package.json

```diff
@@ -167,6 +167,7 @@
 		"@automattic/plans-grid-next": "workspace:^",
 		"@automattic/privacy-toolset": "workspace:^",
 		"@automattic/typography": "workspace:^",
+		"@automattic/ui": "workspace:^",
```

Comment:
I'm not totally sure I follow why we need this defined here and in `client/package.json`, as opposed to just being in `packages/components/package.json`

Reply:
Looks like it was unnecessary, removed in f4e2a82.